sdk/identity/identity/src/credentials/azureCliCredential.ts

```diff
@@ -1,5 +1,7 @@
 // Copyright (c) Microsoft Corporation.
-// Licensed under the MIT License.
+// Licensed under the MIT license.
+
+/* eslint-disable @typescript-eslint/no-useless-constructor */
```

Comment:
If we enable this rule here, we would need to remove the useless constructor, which means a public API change (at least according to api-extractor). Should I do that in this PR? or in another PR? or should I keep this line disabling that rule?

Reply:
It's not that it complains, is that it adds the useless constructor to the file: https://github.com/Azure/azure-sdk-for-js/blame/master/sdk/identity/identity/review/identity.api.md#L51

Though I don't think it matters to remove it, since it's a useless constructor, but I wouldn't do it without getting confirmation from the team :)

Reply:
After speaking with @jonathandturner , he says that it's ok to remove that constructor.